docs/deploy/09-secret-management-and-ingress-controller.md

```diff
@@ -76,6 +76,13 @@ Previously you have configured [workload prerequisites](./08-workload-prerequisi
    az acr import --source docker.io/library/traefik:v3.1 -n $ACR_NAME_AKS_BASELINE
    ```
 
+1. Retrieve the fixed IPv4 address for the AKS internal load balancer ingress controller's service.
```

Comment:
```suggestion
1. When the cluster was deployed, the Bicep deployment decided on the fixed private IP address to use for the AKS internal load balancer ingress controller's service. Retrieve that IP address so we can configure it in the Traefik deployment, which will create the load balancer.
```

Reply:
good catch @johndowns, thanks for the feedback.